pkg/xen-tools/initrd/udhcpc_script.sh

```diff
@@ -81,8 +81,14 @@ case "$1" in
     # configure interface and routes
     ip addr flush dev $interface
     ip addr add ${ip}/${mask} dev $interface
-    # shellcheck disable=SC2086
-    [ -n "$router" ] && [ -n "$staticroutes" ] && install_classless_routes $staticroutes
+    # shellcheck disable=SC2086,SC2154
```

Comment:
Isn't that the wrong thing? Set router on line 7 seems better, no?

Reply:
Sure, thank you